src/Symfony/Bridge/Doctrine/composer.json

```diff
@@ -28,7 +28,7 @@
         "symfony/property-info": "~2.8|3.0",
         "symfony/security": "~2.8|~3.0",
         "symfony/expression-language": "~2.8|~3.0",
-        "symfony/validator": "~2.8|~3.0",
+        "symfony/validator": "~3.1",
```

Comment:
Why wouldn't this test the bridge with symfony/validator anymore? It would simply be tested against 3.1, or am I missing something?